protocols/grpc/agent.proto

```diff
@@ -42,6 +42,7 @@ service AgentService {
 	rpc RemoveInterface(RemoveInterfaceRequest) returns (Interface);
 	rpc AddRoute(AddRouteRequest) returns (google.protobuf.Empty);
 	rpc UpdateRoute(UpdateRouteRequest) returns (google.protobuf.Empty);
+	rpc UpdateRoutes(UpdateRoutesRequest) returns (Routes);
```

Comment:
I am personally fine with this, but I don't see the reason why we would keep `UpdateRoute()` here, seems like it's overlapping.

Reply:
Good point.

Maybe we can keep both APIs temporarily and see if `UpdateRoute` has its own scenario, I think with `UpdateRoutes`, other `route` related APIs including `AddRoute`/`RemoveRoute`/`UpdateRoute` aren't necessary any more.

Reply:
Agreed others should be removed, including updateRoute, imo.